test/lolex-test.js

```diff
@@ -252,24 +250,28 @@ describe("lolex", function () {
         });
 
         it("parses numeric string times", function () {
-            this.clock.setTimeout(function () { lolex.evalCalled = true; }, "10");
+            this.clock.setTimeout(function () { evalCalled = true; }, "10");
             this.clock.tick(10);
 
-            assert(lolex.evalCalled);
+            assert(evalCalled);
         });
 
         it("parses no-numeric string times", function () {
-            this.clock.setTimeout(function () { lolex.evalCalled = true; }, "string");
+            this.clock.setTimeout(function () { evalCalled = true; }, "string");
             this.clock.tick(10);
 
-            assert(lolex.evalCalled);
+            assert(evalCalled);
         });
 
         it("evals non-function callbacks", function () {
-            this.clock.setTimeout("lolex.evalCalled = true", 10);
+            global.evalFn = function () {
+                evalCalled = true;
+            };
+            this.clock.setTimeout("evalFn()", 10);
             this.clock.tick(10);
 
-            assert(lolex.evalCalled);
+            assert(evalCalled);
+            delete global.evalFn;
```

Comment:
Delete this before the assertion so that it's guaranteed to get called. If the assertion above fails, this won't be cleaned up.